scripts/ansible/roles/logstash/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+- name: Add elastic.co's deb signing key 
+  apt_key:
+    url: https://artifacts.elastic.co/GPG-KEY-elasticsearch 
+    state: present
+  become_user: root
+  become: yes 
+- name: Install the package apt-transport-https 
+  apt:
+    name: apt-transport-https
+  become_user: root
+  become: yes
+- name: Install the openjdk package
+  apt:
+    name: openjdk-8-jre
```

Comment:
I heard that this isn't available in ubuntu anymore.

Reply:
I think this is the default now....

Reply:
ah true :+1: